unicode_smilies/unicode_smilies.php

```diff
@@ -685,13 +499,13 @@ function unicode_smilies_smilies(&$a,&$b) {
 	Smilies::add($b, ':baby chick:', '&#x1F424');
 	Smilies::add($b, ':front-facing baby chick:', '&#x1F425');
 	Smilies::add($b, ':bird:', '&#x1F426');
-	Smilies::add($b, ':penguin:', '&#x1F427');
+	Smilies::add($b, ':tux:', '&#x1F427');
```

Comment:
Why this change?

Reply:
The duplicate from version 1.0 was written with Tux. I have adopted the name because it is probably often used out of habit. http://www.unicode.org/emoji/charts/full-emoji-list.html provides for penguin.